app/controllers/concerns/find_common.rb

```diff
@@ -11,7 +11,7 @@ def find_resource
   def resource_finder(scope, id)
     raise ActiveRecord::RecordNotFound if scope.empty?
     result = scope.from_param(id) if scope.respond_to?(:from_param)
-    result ||= scope.friendly.find(id) if scope.respond_to?(:friendly)
+    result ||= scope.friendly.find(id) rescue nil if scope.respond_to?(:friendly)
```

Comment:
Can you write a functional test for a controller that has this problem? (Model?) 

Also if that's the exception that's causing the issue, rescue only that one, we still want to know if something different happened:

```
begin
 result ||= scope.friendly.find(id) if scope.respond_to?(:friendly)
rescue ActiveRecord::ResourceNotFound
end
result || scope.find(id)
```
or similar